packages/cozy-client/src/helpers.js

```diff
@@ -75,3 +75,108 @@ export const generateWebLink = ({
 
   return url.toString()
 }
+
+const uri = (
+  {
+    protocol,
+    hostname,
+    port
+  } /*: { protocol: string, hostname: string, port: string } */
+) /*: string */ => {
```

Comment:
I think this is a flow stuff, here we don't use it. Can you convert it to jsdoc please? ;)

Reply:
Ah right, sorry, old habits :sweat_smile: